packages/react-components/lib/themes/global-theme.tsx

```diff
@@ -2,22 +2,24 @@ import { withStyles } from '@material-ui/core/styles';
 
 export const GlobalCss = withStyles((theme) => ({
   '@global': {
+    // override accordian background to distinguish it from onipanel
     '.MuiAccordion-root': {
       backgroundColor: theme.palette.background.default,
     },
+    // override text color of disabled button to make it visible
     '.MuiButtonBase-root.Mui-disabled': {
       color: theme.palette.text.primary,
     },
-    '.MuiAppBar-root': {
-      backgroundColor: theme.palette.primary.main,
-    },
+    // highlight table header in panels
```

Comment:
Whats the reasoning to change this? It doesn't seem to help to make the table head more prominent in rmf-dark.

Reply:
The header looks like it was blending into the background and I thought this would make it look better. Removed it for now.